project-base/storefront/components/Pages/Search/SearchProductsContent.tsx

```diff
@@ -0,0 +1,88 @@
+import { Pagination } from 'components/Blocks/Pagination/Pagination';
+import { ProductsList } from 'components/Blocks/Product/ProductsList/ProductsList';
+import { SkeletonModuleProductListItem } from 'components/Blocks/Skeleton/SkeletonModuleProductListItem';
+import { DEFAULT_PAGE_SIZE } from 'config/constants';
+import { SearchProductsQueryApi } from 'graphql/generated';
+import { useGtmPaginatedProductListViewEvent } from 'gtm/hooks/productList/useGtmPaginatedProductListViewEvent';
+import { GtmMessageOriginType, GtmProductListNameType } from 'gtm/types/enums';
+import { createEmptyArray } from 'helpers/arrayUtils';
+import { getMappedProducts } from 'helpers/mappers/products';
+import Trans from 'next-translate/Trans';
+import useTranslation from 'next-translate/useTranslation';
+import { RefObject } from 'react';
+
+type SearchProductsContentProps = {
+    isFetching: boolean;
+    isLoadMoreFetching: boolean;
+    paginationScrollTargetRef: RefObject<HTMLDivElement>;
+    searchProductsData: SearchProductsQueryApi['productsSearch'];
+};
+
+export const SearchProductsContent: FC<SearchProductsContentProps> = ({
+    isFetching,
+    isLoadMoreFetching,
+    paginationScrollTargetRef,
+    searchProductsData,
+}) => {
+    const { t } = useTranslation();
+    const searchResultProducts = getMappedProducts(searchProductsData.edges);
+
+    useGtmPaginatedProductListViewEvent(searchResultProducts, GtmProductListNameType.search_results);
+
+    const isWithProductsShown = !!searchProductsData.totalCount;
+    const noProductsFound = parseInt(searchProductsData.productFilterOptions.maximalPrice) === 0;
```

Comment:
This might have been here before and would thus make sense to keep it. Just thought about a edge scenario, where there are "free" products you can add to your cart, and this would make them invisible. If all you search for is a free product, it will not be visible. Just a thought, if you want to investigate or not.

Reply:
I have no idea about business behind this but I assume this was tested and validated.

Reply:
I will resolve this here and just leave it as a comment for BV.